tests/command/test_amend.rs

```diff
@@ -0,0 +1,193 @@
+use branchless::testing::{make_git, GitRunOptions};
+
+use crate::command::test_restack::remove_rebase_lines;
+
+#[test]
+fn test_amend_with_children() -> eyre::Result<()> {
+    let git = make_git()?;
+
+    if !git.supports_committer_date_is_author_date()? {
+        return Ok(());
+    }
+
+    git.init_repo()?;
+    git.run(&["config", "branchless.restack.preserveTimestamps", "true"])?;
```

Comment:
`branchless.restack.preserveTimestamps` should be the default as of #167, so no need to set it explicitly here.